src/components/10-atoms/progress-bar/index.js

```diff
@@ -0,0 +1,82 @@
+import { LitElement, html, css, unsafeCSS } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
+import { defineVersioned } from '../../../utils/component-versioning';
+import { applyDefaults } from '../../../utils/with-react';
+import styles from './index.scss';
+
+class AXAProgressBar extends LitElement {
+  static get tagName() {
+    return 'axa-progress-bar';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      small: { type: Boolean, defaultValue: false },
+      fullWidth: { type: Boolean, defaultValue: false },
+      value: { type: Number },
+      max: { type: Number },
+      text: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    applyDefaults(this);
+  }
+
+  calculatePercantage() {
+    if (this.max !== 0) {
+      if (this.value > this.max) {
+        return 100;
+      }
+      return (this.value / this.max) * 100;
+    } else if (this.value > 100) {
+      return 100;
+    }
+    return this.value;
+  }
+
+  render() {
+    const classesBorder = {
+      'a-progress-bar__small': this.small,
+      'a-progress-bar__full-width': this.fullWidth,
+    };
+
+    const classesLoader = {
+      'a-progress-bar__small': this.small,
+      'a-progress-bar__full-width-loader': this.fullWidth,
+      'a-progress-bar__full-width-loader-max':
+        this.fullWidth && this.value >= 100,
+    };
+
+    return html`
+      <article>
+        <div class="a-progress-bar__border ${classMap(classesBorder)}">
+          <div
+            class="a-progress-bar ${classMap(classesLoader)}"
```

Comment:
`a-progress-bar__progress-bar`

Would be correct, even though it looks weird. Because `a-progress-bar` would be a parent of `a-progress-bar__border`, but the opposite is the case.